webapp-django/crashstats/supersearch/form_fields.py

```diff
@@ -64,7 +65,7 @@ def value_to_string(self, value):
         """Return the value as a string. """
         if value is None:
             return None
-        return unicode(value)
+        return str(value)
```

Comment:
This should probably be `six.text_type`. It needs to be `unicode` in Python 2, but `str` in Python 3.